aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -239,6 +239,13 @@ module aptos_framework::coin {
         exists<CoinInfo<CoinType>>(coin_address<CoinType>())
     }
 
+    #[view]
+    /// Returns `true` is account_addr has frozen the CoinStore
+    public fun is_coin_store_frozen<CoinType>(account_addr: address): bool acquires CoinStore {
+        let coin_store = borrow_global<CoinStore<CoinType>>(account_addr);
```

Comment:
check `is_account_registered`?

Reply:
Maybe this should be just is frozen?

Reply:
Is_frozen can be confusing because coin::is_frozen gives the impression that the coin is frozen?

Reply:
I believe `is_coin_store_frozen` name describes in accuracy what the function returns. From the caller moule perspective, this information is needed either when calling `withdraw` or `deposit` because both functions have the following assertion:

```
assert!(
  !coin_store.frozen,
  error::permission_denied(EFROZEN),
);
```

So knowing whether the coin_store is frozen can help caller decide what to do. Or maybe for the same exact reason we can call the function `is_coin_store_active`?

@lightmark I believe you're right, we should probably check `is_account_registered ` instead of aborting (btw how does abort work with #view functions?).